neuralmonkey/config/configuration.py

```diff
@@ -67,11 +63,40 @@ def load_file(self, path):
             log("Failed to load INI file: {}".format(exc), color='red')
```

Comment:
Mě teda fascinuje, že máš odvahu o kusu Pythonového kódu tvrdit, že tam může nastat jenom nějaký typ výjimky. A v tomhle případě to triviálně není pravda – ten file se tam otevírá a parsuje.